iron-menu-behavior.html

```diff
@@ -262,16 +268,16 @@
      * @param {CustomEvent} event A key combination event.
      */
     _onShiftTabDown: function(event) {
-      var oldTabIndex = this.getAttribute('tabindex');
+      var couldTab = this.getAttribute('tabindex') == 0;
```

Comment:
Done. I had used == b/c we aren't setting as anything else. I didn't think about a user setting the tabindex to anything else. If a user manually sets the tab index, it'll be lost.... Is that an issue?